### PR TITLE
docs: Update TypeScript plugin section for VS Code prompt

### PR DESCRIPTION
**Pull request by jenewland1999**
Hey Next.js Team, 👋🏻

I noticed Next.js apps scaffolded with create-next-app weren't prompting me for the TypeScript workspace version in VS Code. After digging through the repo, I stumbled on this https://github.com/vercel/next.js/pull/49133 PR from Tim. While this PR explains why it was removed, I'm not sure the docs were ever updated to reflect this change.

This PR aims to address that with a simple note in the TypeScript Plugin section of the TypeScript docs page.

### CODE REVIEW

File: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx [hunk 1]
```diff
@@ -59,7 +49,11 @@ If you miss the prompt, you can enable the plugin manually by:
   height="637"
 />
 
-Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used. Further, we automatically create a VSCode settings file for you to automate this process.
+Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used.
+
+> **Good to know:**
+> 
+> Prior to v13.4.0 VS Code would prompt users to set the TypeScript version when running `npm run dev` for the first time. However, this was removed until such time a better solution is implemented.
```
Comment (delbaoliveira) [inline docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx:56]:
```suggestion
> Prior to v13.4.0, VS Code would prompt users to enable the plugin when running `npm run dev` for the first time. However, this caused issues for some users and was removed until a better solution is implemented. Please follow the steps above to manually enable the plugin. 
```

I realise "prompt users to set the TypeScript version" is strictly accurate, but the secondary effect is what users need to know here IMO :pray:

**Review by delbaoliveira — CHANGES_REQUESTED** [review]
Thank you!

**Review by delbaoliveira — DISMISSED** [review]

File: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx [hunk 2]
```diff
@@ -35,17 +35,7 @@ Add TypeScript to your project by renaming a file to `.ts` / `.tsx`. Run `next d
...
-<Image
-  alt="TypeScript Prompt"
-  srcLight="/docs/light/typescript-prompt.png"
-  srcDark="/docs/dark/typescript-prompt.png"
-  width="1600"
-  height="403"
-/>
-
-If you miss the prompt, you can enable the plugin manually by:
+You can enable the plugin in VSCode by:
```
Comment (leerob) [inline docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx:38]:
```suggestion
You can enable the plugin in VS Code by:
```

File: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx [hunk 3]
```diff
@@ -59,7 +49,11 @@ If you miss the prompt, you can enable the plugin manually by:
   height="637"
 />
 
-Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used. Further, we automatically create a VSCode settings file for you to automate this process.
+Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used.
+
+> **Good to know:**
+>
+> Prior to v13.4.0, VS Code would prompt users to enable the plugin when running `npm run dev` for the first time. However, this caused issues for some users and was removed until a better solution is implemented. Please follow the steps above to manually enable the plugin.
```
Comment (leerob) [inline docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx:56]:
```suggestion
```

I actually don't think we need to say this at all.

File: docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx [hunk 4]
```diff
@@ -59,7 +49,10 @@ If you miss the prompt, you can enable the plugin manually by:
   height="637"
 />
 
-Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used. Further, we automatically create a VSCode settings file for you to automate this process.
+Now, when editing files, the custom plugin will be enabled. When running `next build`, the custom type checker will be used.
+
+> **Good to know:**
+>
```
Comment (leerob) [inline docs/02-app/01-building-your-application/06-configuring/01-typescript.mdx:55]:
```suggestion
```

**Review by leerob — APPROVED** [review]

**Pull request merged**